### ISSUE TITLE
tests fail when seeded with 13277 or 22042

### ISSUE THREAD

**Issue by paulkogel** [devnacho/mountain_view#34]
when running `rake TESTOPTS="--seed=13277"` on current master i get this failure (ruby 2.3.0, OSX 10.11.3):

```
Run options: --seed 13277

# Running:
.F......................
Finished in 0.544574s, 44.0711 runs/s, 145.0674 assertions/s.

  1) Failure:
ComponentGeneratorTest#test_Assert_all_files_are_properly_created [/Users/paul/p/fejo/mountain_view/test/generators/component_generator_test.rb:12]:
Expected file "app/components/widget/_widget.html.erb" to exist, but does not

24 runs, 79 assertions, 1 failures, 0 errors, 0 skips
rake aborted!
Command failed with status (1): [ruby -w -I"lib:lib:test" -I"/Users/paul/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/rake-11.1.1/lib" "/Users/paul/.rbenv/versions/2.3.0/lib/ruby/gems/2.3.0/gems/rake-11.1.1/lib/rake/rake_test_loader.rb" "test/**/*_test.rb" ]

Tasks: TOP => default => test
(See full trace by running task with --trace)
```

**Comment by OddEssay** [comment]
I can reproduce the same error with ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin14] and the seed specified in `rake TESTOPTS="--seed=13277"`

**Comment by kitop** [comment]
I can reproduce too. Very good catch! Thanks!!
I'll investigate why it happens.

**Comment by kitop** [comment]
Fixed in 76d9516 :)
